### DISCUSSION TITLE
Implementation of Relational Data Base Store with IDataStore

### DISCUSSION THREAD

**Discussion by ayuksekkaya** [jezzsantos/saastack#43] [Ideas]
I think it could be a good idea to add an implementation of RelationalDatabaseDataStore example in code alongside InProcess and LocalMachine. It would make the code more production ready. I am a bit confused on if I should implement that using EF core and how that would work with the CommandEntity Structure. I see some references to AzureSqlDataStore in the documentation but I couldn't find a file named AzureSqlDataStore. Was it there before and it got removed for some reason?

**Comment by jezzsantos** [comment]
Thanks @ayuksekkaya 

You are right about that.

We do plan to have the following stores very soon - next couple weeks. (see https://github.com/jezzsantos/saastack/issues/4#issue-1886843428)

To goal is to get all these as soon as I can for production workloads:
* Azure:
  * `SQLServerDataStore`
  * `AzureTableStorageDataStore`
  * `AzureCosmosSqlDataStore`
  * `AzureCosmosTableDataStore`
* AWS:
  * `RDSDbDataStore`
  * `DynamoDbDataStore`
* Other:
  * `RedisDataStore`
  * `SQLServerEventStore`
  * `EventStoreEventStore`

It may take some time to get them all.

**Comment by jezzsantos** [comment]
@ayuksekkaya `AzureSqlServerStore` is now in the codebase, ready for production workloads: 29443dce4bc968e883fe311a2500b99b4c3bcc9f

**Reply by ayuksekkaya** [reply]
Thank you @jezzsantos !
